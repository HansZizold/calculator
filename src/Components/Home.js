import React from 'react';
import styles from './Home.module.css';

function Home() {
  return (
    <>
      <h2 className={styles.title}>Welcome to our page!</h2>
      <p>
        Lorem ipsum dolor sit amet consectetur adipisicing elit. Sapiente animi
        quod alias iste dicta tempore id est magni, ducimus cum distinctio quo.
        Perferendis nemo minus alias earum consectetur nostrum sint molestiae,
        deserunt tempore et dolorum omnis hic praesentium eos aliquid
        architecto iure, maiores sunt necessitatibus distinctio. Provident
        ducimus nostrum et!
      </p>
      <br />
      <p>
        Lorem ipsum dolor sit amet consectetur adipisicing elit. Error facere,
        dignissimos debitis, ipsam praesentium impedit totam sequi, quia commodi
        possimus temporibus! Molestias porro nihil perferendis quasi nesciunt
        recusandae adipisci libero, dignissimos culpa quidem sit alias sequi sed
        fugit maxime iure, accusantium, veritatis ea. Impedit fugit vitae ad
        accusamus, facere voluptas.
      </p>
    </>
  );
}
export default Home;
