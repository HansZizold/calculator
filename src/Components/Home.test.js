import React from 'react';
import { render, screen } from '@testing-library/react';
import renderer from 'react-test-renderer';
import Home from './Home';

describe('Home Page Tests', () => {
  test('Check the title of the page', () => {
    render(<Home />);
    expect(screen.getByText('Welcome to our page!')).toBeTruthy();
  });

  test('Quote renders correctly', () => {
    const tree = renderer.create(<Home />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
