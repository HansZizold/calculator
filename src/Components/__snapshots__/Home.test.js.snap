// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home Page Tests Quote renders correctly 1`] = `
Array [
  <h2
    className="title"
  >
    Welcome to our page!
  </h2>,
  <p>
    Lorem ipsum dolor sit amet consectetur adipisicing elit. Sapiente animi quod alias iste dicta tempore id est magni, ducimus cum distinctio quo. Perferendis nemo minus alias earum consectetur nostrum sint molestiae, deserunt tempore et dolorum omnis hic praesentium eos aliquid architecto iure, maiores sunt necessitatibus distinctio. Provident ducimus nostrum et!
  </p>,
  <br />,
  <p>
    Lorem ipsum dolor sit amet consectetur adipisicing elit. Error facere, dignissimos debitis, ipsam praesentium impedit totam sequi, quia commodi possimus temporibus! Molestias porro nihil perferendis quasi nesciunt recusandae adipisci libero, dignissimos culpa quidem sit alias sequi sed fugit maxime iure, accusantium, veritatis ea. Impedit fugit vitae ad accusamus, facere voluptas.
  </p>,
]
`;
